syntaxes/brightscript.tmLanguage.json

```diff
@@ -781,7 +797,7 @@
             }
         },
         "interface_field": {
-            "begin": "(?i)\\s*\\b([a-z0-9_]+)(?:[\\s\\t]*(as))?",
+            "begin": "(?i)\\s*\\b([a-z0-9_]+)(?:[\\s\\t]*(as))",
```

Comment:
I think we might want to add  `optional` in this regex as an additional optional capture group rather than capturing it as its unique named item.